### PR TITLE
I/OSERDES BEL timing

### PR DESCRIPTION
**Pull request by litghost**
This only modified the tcl to change the output, but it immediately causes the post-processing script to fail.  @kgugala Can you see if you work out what is up?  The post-processing script needs some work, as it is currently very hard to understand or maintain.

### CODE REVIEW

**Comment by litghost** [comment]
@kgugala I was able to refactor the script, still debugging additional steps.

**Comment by litghost** [comment]
@acomodi FYI

**Comment by kgugala** [comment]
the postprocessing script is sensitive to tcl output format. I'll take a look on that. Probably need better format documentation.

**Comment by litghost** [comment]
> the postprocessing script is sensitive to tcl output format. I'll take a look on that. Probably need better format documentation.

To be clear, I believe I've fixed the script at this point.  I've separated the parsing from the processing, which helps understand the processing logic.  I've also accommodated repeated tile instances, which is required if a tile has "alternate site types".  There is likely a way to automate the TCL change, but for now this will get us started.

File: fuzzers/007-timing/bel/Makefile [hunk 1]
```diff
@@ -19,6 +19,7 @@ build/fixup_timings: build/run.ok
 	python3 fixup_timings_txt.py --txtin build/bel_pins.txt --txtout build/bel_pins.txt --site RAMBFIFO36E1 --slice BRAM_R --type pins
 	python3 fixup_timings_txt.py --txtin build/bel_properties.txt --txtout build/bel_properties.txt --site RAMBFIFO36E1 --slice BRAM_L --type properties
 	python3 fixup_timings_txt.py --txtin build/bel_properties.txt --txtout build/bel_properties.txt --site RAMBFIFO36E1 --slice BRAM_R --type properties
+	touch build/fixup_timings
```
[the remark this answers is not in the document]
Comment (litghost) [reply fuzzers/007-timing/bel/Makefile]: > Is `fixup_timings_txt.py` idempotent?

Yes, it is idempotent.

**Comment by litghost** [comment]
> I think this probably needs someone from antmicro to review like @kgugala ?

I already asked for a review and didn't get one :/

**Pull request merged**